### DISCUSSION TITLE
Drive Space Indicator

### DISCUSSION THREAD

**Discussion by Ansh4011** [aleksey-hoffman/sigma-file-manager#69] [Ideas]
Feature request: Drive Space Indicator can be a given a new colour if possible bcz the the green one kinda camouflages with the dark background! My recommendation is for a pop purple gradient! I have given images for the reference too! This feature can be addressed in future though! Thanks!
![image](https://user-images.githubusercontent.com/84930340/119949545-997da200-bfb7-11eb-8329-38df345c6597.png)
![image](https://user-images.githubusercontent.com/84930340/119949564-9d112900-bfb7-11eb-9c7b-e9c298e094be.png)

**Comment by InstantCassette** [comment]
Ideally this would be part of an overall theme manager where you can customize base & accent colors. [Also linked to the Light Theme](https://github.com/aleksey-hoffman/sigma-file-manager/discussions/74)

I would also love to customize the icon and a color, that way you can easily identify groups of drives. ( I have loads of unraid mapped network locations. )

**Reply by Ansh4011** [reply]
> Ideally this would be part of an overall theme manager where you can customize base & accent colors. [Also linked to the Light Theme](https://github.com/aleksey-hoffman/sigma-file-manager/discussions/74)
> 
> I would also love to customize the icon and a color, that way you can easily identify groups of drives. ( I have loads of unraid mapped network locations. )

Agreed! It will really make identification easy and fast.
